ambari-agent/src/main/python/ambari_agent/AmbariConfig.py

```diff
@@ -47,9 +47,10 @@
 data_cleanup_max_age=2592000
 data_cleanup_max_size_MB = 100
 ping_port=8670
-cache_dir={ps}var{ps}lib{ps}ambari-agent{ps}cache
+cache_dir={ps}tmp
 parallel_execution=0
 system_resource_overrides={ps}etc{ps}resource_overrides
+tolerate_download_failures=false
```

Comment:
Why is this defaulted false?  I would think higher tolerance would be appropriate "out of the box" so to speak.

Reply:
This code is used only for Python tests. So this won't affect real clusters. (sadly we have this kind of hack for real long time)